src/process/process.rs

```diff
@@ -63,8 +63,9 @@ pub struct Callback {
     pub pc: usize
 }
 
-#[repr(C,packed)]
```

Comment:
yeah you can do that as a no-op with `transmute`. See the new impl for `static_init!`